frontend/src/components/ViewActivities/activitylist.js

```diff
@@ -35,7 +35,7 @@ export async function getActivityList(tripId) {
         tripActivities.push(data);
       })
     }).catch(error => {
-      console.log('It seems that an error has occured.');
+      console.log('It seems that an error has occured.', error);
```

Comment:
Nit, use console.error?
```suggestion
      console.error('It seems that an error has occured.', error);
```

Reply:
I didn't know that was a thing! thanks.